apidoc/Titanium/Media/VideoPlayer.yml

```diff
@@ -598,47 +556,35 @@ properties:
     permission: read-only
 
   - name: media
-    summary: Media object to play, as either a `File`,  a `Blob`, or a URL.
+    summary: Media object to play, as either a `Ti.Filesystem.File`, a `Ti.Blob`, or an URL `String`.
     platforms: [iphone, ipad, macos]
     type: [Titanium.Blob, Titanium.Filesystem.File, String]
     permission: write-only
 
   - name: mediaControlStyle
-    exclude-platforms: [macos]
+    platforms: [android]
     deprecated:
         since: "7.0.0"
-        notes: On iOS, use [Titanium.Media.VideoPlayer.showsControls](Titanium.Media.VideoPlayer.showsControls) instead.
-        removed: "7.0.0"
+        notes: Use [Titanium.Media.VideoPlayer.showsControls](Titanium.Media.VideoPlayer.showsControls) instead.
     summary: The style of the playback controls.
-    description: |
-        Mobile web only supports [VIDEO_CONTROL_NONE](Titanium.Media.VIDEO_CONTROL_NONE) and
-        [VIDEO_CONTROL_DEFAULT](Titanium.Media.VIDEO_CONTROL_DEFAULT).
+    description: Removed from iOS in SDK 7.0.0
     type: Number
     constants: Titanium.Media.VIDEO_CONTROL_*
     default: System default video controls (<Titanium.Media.VIDEO_CONTROL_DEFAULT>).
 
   - name: mediaTypes
```

Comment:
As per code, I can see this is read-only property.  Need to add `permission: read-only`.